packages/babel-polyfill/src/index.js

```diff
@@ -5,3 +5,4 @@ global._babelPolyfill = true;
 
 import "core-js/shim";
 import "regenerator-runtime/runtime";
+import "./bigint/bigint";
```

Comment:
I don't think we want to add this to babel-polyfill itself, since most people won't be using it and will ask about the size increase.